tests/runtime_test.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import pytest
+import sys
+import os
+
+
+def main():
+    if sys.version_info < (3, 0):
+        version_dirs = [
+            '2and3',
+            '2',
+        ]
+    else:
+        version_dirs = [
+            '2and3',
+            '3',
+            '%d.%d' % (sys.version_info[0], sys.version_info[1]),
```

Comment:
@JelleZijlstra I believe it doesn't apply to run-time checks since we run them on a particular version of the interpreter and it may not have some module from older Python versions.